drivers/storage/portworx/components_test.go

```diff
@@ -321,6 +354,19 @@ func TestBasicComponentsInstall(t *testing.T) {
 	require.Equal(t, expectedPXService.Labels, pxService.Labels)
 	require.Equal(t, expectedPXService.Spec, pxService.Spec)
 
+	// Portworx Service Account Token Secret
+	expectedPXSaTokenSecret := testutil.GetExpectedSecret(t, "portworxServiceAccountTokenSecret.yaml")
+	saTokenSecret := &v1.Secret{}
+	err = testutil.Get(k8sClient, saTokenSecret, pxutil.PortworxServiceAccountTokenSecretName, cluster.Namespace)
+	require.NoError(t, err)
+	fmt.Println(saTokenSecret)
+	require.Equal(t, expectedPXSaTokenSecret.Name, saTokenSecret.Name)
+	require.Equal(t, expectedPXSaTokenSecret.Namespace, saTokenSecret.Namespace)
+	require.Len(t, saTokenSecret.OwnerReferences, 1)
+	require.Equal(t, cluster.Name, saTokenSecret.OwnerReferences[0].Name)
+	require.Equal(t, len(expectedPXSaTokenSecret.Data), len(saTokenSecret.Data))
```

Comment:
we could also return a specific token from the mock and verify that it is set in the secret here?